### ISSUE TITLE
Properties created via the Object Type Editor do not get exported

### ISSUE THREAD

**Issue by ashwin911** [Seanba/Tiled2Unity#31]
If I create a new type with a property in the Object Type Editor and add that type to an object, I can see the property in the properties window of that object. But when I export it, only properties that have been created per object in the Properties window appear in the XML file.

**Comment by Seanba** [comment]
Could you send me an example TMX file? Sometimes there are Tiled features
I'm not aware of and this might be one of them.

Best,
Sean

**Comment by ashwin911** [comment]
I've added a fireOn property in the Object Types Editor. I think I know what the problem is. The properties created in Object Types Editor aren't written to the TMX file but need to be exported separately to a XML file and reimported if you open the TMX map on a different computer. I have included that xml file in the zip file.

[objectTypeEditor export test.zip](https://github.com/Seanba/Tiled2Unity/files/184502/objectTypeEditor.export.test.zip)
